Jenkinsfile

```diff
@@ -1,6 +1,6 @@
-@Library('defra-shared@master') _
+@Library('defra-shared@iwtf-3523-codeartifact') _
```

Comment:
I'm sure it goes without saying, but when https://gitlab-dev.aws-int.defra.cloud/jenkins/jenkins-shared/-/merge_requests/3 is merged, this should change to defra-shared@master. This needs to be done before this PR is merged

Reply:
See comment above :)